libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -4801,8 +4801,27 @@ static void link_physical_logical_types(std::vector<t_physical_tile_type>& Physi
 
         for (int pin = 0; pin < logical_block.pb_type->num_pins; pin++) {
             for (auto& tile : logical_block.equivalent_tiles) {
-                auto direct_map = tile.tile_block_pin_directs_map[logical_block.index];
```

Comment:
This diff hunk probably should've been in the previous commit

Reply:
Done